front/reservation.form.php

```diff
@@ -113,7 +113,7 @@
             $input['users_id'] = $_POST['users_id'];
```

Comment:
maybe it would be better `$input['users_id'] = $_POST['users_id'];` and keep the `===`?

Reply:
maybe the same for user.class